0005-on-chain-voting-transaction-specification.proto

```diff
@@ -0,0 +1,24 @@
+syntax = "proto3";
+
+// PeerAssets voting protocol specification
+// written by peerchemist <peerchemist@protonmail.ch>
+
+// Identity message
+message Vote {
+
+  // voting protocol version number
+  uint32 version = 1;
+
+  // Modes for card voting
+  enum MODE {
```

Comment:
Maybe it can even be like below, as it seems the WEIGHT codes are multiplications of SIMPLE (adding a multiplier):
```
    NONE = 0x00
    SIMPLE = 0x01; // One vote per transaction
    WEIGHT_CARD_BALANCE = 0x03; // (0x01 | 0x02) Weightning vote with PA card balance
    WEIGHT_CARD_DAYS = 0x07; //  (0x01 | 0x02 | 0x03) Weightnig vote with PA card days
```

implementation can then be as simple as
```
if (mode & SIMPLE)
   vote = 1;
if (mode & DAYS)
  vote *= nr_of_days;
if (mode & DAYS)
  vote *= balance;
```